Kickstarter-iOS/Views/Cells/SettingsFollowCell.swift

```diff
@@ -50,7 +50,7 @@ internal final class SettingsFollowCell: UITableViewCell, ValueCell {
       .observeForUI()
       .observeValues { [weak self] in
         guard let _self = self else { return }
```

Comment:
For interest sake, @dusi and I have started using:
```swift
guard let `self` = self else { return }
```
That way the subsequent line can refer to:
```swift
self.delegate?.settingsFollowCellDidDisableFollowing(self)
```
and the compiler is satisfied. The backticks around `self` just allows one to use a reserved keyword in local scope.

Reply:
Yeah I've used that in the past too, but for consistency continued using the `_self` pattern since that's what we use everywhere else.